src/python/analyzer_executor/src/analyzer_executor_lib/analyzer_executor.py

```diff
@@ -353,7 +362,7 @@ def exec_analyzer(
                             f"Execution of {name} failed with {e} {e.args}"
                         )
                         sender.send(ExecutionFailed())
-                        raise
+                        raise e
```

Comment:
Fun fact - there is a difference here, and I think the previous way is more in line with what we want.

`raise` will reraise the exception with its original traceback
`raise e` will reraise the exception without its original traceback, instead saying the exception originated from analyzer_executor.py:365

Reply:
wow that's unexpected, will fix

Reply:
fixed